Actuators   actuators     # control actuators (gas, brake and steer)
Calibration calibration   # calibration control
bool        activate_auto # true indicates that the user requested activating auto mode
